aerospike-storage-backend/src/main/java/com/playtika/janusgraph/aerospike/operations/BasicScanOperations.java

```diff
@@ -46,9 +46,13 @@ public KeyIterator getKeys(String storeName, SliceQuery query, StoreTransaction
             try {
                 aerospikeOperations.getClient().scanAll(scanPolicy,
                         aerospikeOperations.getNamespace(), aerospikeOperations.getSetName(storeName), keyIterator);
+                logger.info("Finished scan operation storeName=[{}], query=[{}], tx=[{}]", storeName, query, txh);
```

Comment:
are you sure that info is okay here?

Reply:
yes, scan is quite rare (rather exceptional) operation for janusgraph so we may afford to write it with info level